scheduler/src/cook/kubernetes/api.clj

```diff
@@ -156,7 +156,7 @@
   "Converts a map of Kubernetes resources to a cook resource map {:mem double, :cpus double}"
   [m]
   {:mem (if (get m "memory")
-          (-> m (get "memory") to-double (/ (* 1024 1024)))
+          (-> m (get "memory") to-double (/ 1000000))
```

Comment:
Please put in a comment as to why.
"Cook tracks memory in megabytes (1000*1000) bytes. Convert kubernetes memory into megabytes."